docs/app/showcase/charts/dist/app/views/socialChart/templates/edgeDetails.html

```diff
@@ -2,7 +2,7 @@
 
 <div class="left-node social-edge-layout-demo social-edge-demo-clickable" ng-click="sc.selectedEdge.sourceNode.goTo()">
 	<div class="social-edge-demo-inner">
-		<i class="hpe-icon hpe-user" style="color:#2ad2c9; font-size:20px;"></i>
+		<i class="hpe-icon hpe-user social-edge-demo-icon"></i>
```

Comment:
Check the social chart documentation page for how we solved this previously. By using the text-vibrant1 class, we make this theme-independent. This will be important if we host an HPE theme version again, and when the Microfocus branding occurs.